### PR TITLE
Headless getting started

### PR DESCRIPTION
**Pull request by EmilK15**
Added some revisions to the Getting Started portion of the documentation regarding the addition of create-webiny-project, the new webiny deploy api name, and added a note so that if a user wants to change the name of a folder they can still run their commands the same way with the new folder name.

### CODE REVIEW

**Comment by Pavel910** [comment]
@EmilK15 Just realized we don't have Netlify build set up for `next` branch.

File: docs/get-started/going-live.md [hunk 1]
```diff
@@ -49,17 +49,18 @@ However, if you want to use a different database, for example, you can create a
...
 ## 3. Deploy apps using `dev` environment
 
 From the root of your project, run the following command:
 
 ```
-webiny deploy-apps --env=dev
+webiny deploy apps --env=dev
 ```
 
+>Note: if you rename the `apps` folder to something else like `src` you can simply use `webiny deploy src --env=dev`
```
Comment (doitadrian) [inline docs/get-started/going-live.md:63]: image

1. I would reword the note below this snippet, and rather point out that the `apps` actually references the folder in which the deployment configuration (`resources.js`) is located. Which means that, if needed, user can create his own additional folders if needed, for example `api-public`, `apps-intranet`, etc.

2. Small error with font.

File: docs/get-started/quick-start.md [hunk 2]
```diff
@@ -32,42 +32,51 @@ The following things are mandatory for both local development and production dep
 
 > NOTE: if you're unsure your AWS credentials are configured correctly, you can verify them by running the following command using [AWS CLI](https://aws.amazon.com/cli/): `aws sts get-caller-identity`. If you don't see the user info, take a look at this tutorial to [create an IAM user for programmatic usage](https://www.youtube.com/watch?v=tgb_MRVylWw).
 
-## 1. Install the CLI
+## 1. Create a new project
 
 ```
-npm -g install @webiny/cli
+npx create-webiny-project myProject --template templateName
```
Comment (doitadrian) [inline docs/get-started/quick-start.md:38]: I would rather put the final command here. In other words, I'd put the `full` for the `--template` (instead of the `templateName`). That way, users can quickly just c/p the command. And of course, we can then reword the tip below the command snippet accordingly.

File: docs/get-started/quick-start.md [hunk 3]
```diff
@@ -32,42 +32,51 @@ The following things are mandatory for both local development and production dep
...
 
 Edit `.env.json` file in the root of the project and set the `MONGODB_SERVER` value. The values in this file are unique for your project, you do NOT need to change any other values.
 
 > In case you need any help creating a MongoDB cluster on MongoDB Atlas, please follow [this guide](/docs/other/guides/mongodb-atlas).
 
 > IMPORTANT: it's important to give the outside world access to your database because the database will be accessed from your cloud functions, thus you'll never have a fixed IP address. See the [Whitelist Your Connection IP Address](https://docs.atlas.mongodb.com/getting-started/#whitelist-your-connection-ip-address). Make sure you add a `0.0.0.0/0` entry.
 
-> The `MONGODB_SERVER` value should be in the format of a MongoDB connection string such as:
-> `mongodb+srv://{YOUR_USERNAME}:{YOUR_PASSWORD}@someclustername.mongodb.net`.
+Your `.env.json` file should look something like this after updating your `MONGODB_SERVER` and `MONGODB_NAME` parameters.
```
Comment (doitadrian) [inline docs/get-started/quick-start.md:51]: Maybe it's just me, but I like putting `:` when I'm about to present a code snippet. So, I might put something like:
> After updating the `MONGODB_SERVER` and `MONGODB_NAME` variables, you might end up with something like the following:

Comment (doitadrian) [reply docs/get-started/quick-start.md:51]: But in the end, leave what you feel is best.

File: docs/get-started/quick-start.md [hunk 4]
```diff
@@ -32,42 +32,51 @@ The following things are mandatory for both local development and production dep
...
+## 3. Deploy API
 
 We need to deploy a `local` API environment to use for local development:
 
 ```
-webiny deploy-api
+webiny deploy api --env=local
 ```
 
+If you were to rename the folder you want to deploy your api from to `server` you would just need to run `webiny deploy server` instead.
```
Comment (doitadrian) [inline docs/get-started/quick-start.md:74]: Same comment as in the `docs/get-started/going-live.md`.

File: docs/get-started/quick-start.md [hunk 5]
```diff
@@ -89,6 +98,8 @@ cd apps/site
 yarn start
 ```
 
+> NOTE: you can also run `webiny deploy apps --env=local` to run every folder within the `apps` folder.
```
Comment (doitadrian) [inline docs/get-started/quick-start.md:101]: `to run every folder` ?

You mean `in order to deploy all apps within ...` ?

File: docs/get-started/quick-start.md [hunk 6]
```diff
@@ -89,6 +98,8 @@ cd apps/site
...
+
 ---
 
 This is it! You have deployed a `local` API environment, and can begin developing your React apps on your local machine. For deploys to production and other environments, please see the next page.
```
Comment (doitadrian) [inline docs/get-started/quick-start.md:105]: Let's put a link on the word `next`.
